pkgs/misc/doge/default.nix

```diff
@@ -0,0 +1,21 @@
+with import <nixpkgs> {};
```

Comment:
Instead of importing `<nixpkgs>` you should have a function declaration, like: `{ stdenv, python3Packages, etc }`

And then you need to add it to `all-packages.nix`.

Reply:
done